docs/pages/router/layouts/settings.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Settings
```

Comment:
```suggestion
title: Router Settings
sidebar_title: Settings
```

Many of Router pages will need to be renamed for better searchability and SEO purpose, we cannot stick to generic terms, since this could lead to unnecessary confusion (already have that problem with FAQ etc.).